libs/components/theme/src/lib/theming/theme.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { SkyTheme } from './theme';
+import { SkyThemeMode } from './theme-mode';
+import { SkyThemeSpacing } from './theme-spacing';
+
+describe('SkyTheme', () => {
+  it('should default supportedSpacing to an empty array when not specified', () => {
+    const theme = new SkyTheme('foo', 'sky-theme-foo', [
+      SkyThemeMode.presets.light,
+    ]);
+
+    expect(theme.supportedSpacing).toEqual([SkyThemeSpacing.presets.standard]);
```

Comment:
the test says it should default to an empty array but that doesn't look like an empty array?